libcxx/include/string

```diff
@@ -749,6 +749,32 @@ struct __can_be_converted_to_string_view
 struct __uninitialized_size_tag {};
 struct __init_with_sentinel_tag {};
 
+template <size_t _PaddingSize>
+struct __padding {
+  char __padding_[_PaddingSize];
+};
+
+template <>
+struct __padding<0> {};
+
+template <class _CharT, size_t __min_cap, size_t _Padding = sizeof(_CharT) - 1>
```

Comment:
This shouldn't be required to be a template anymore.

Reply:
Indeed!